Refresh.GameServer/Types/Visibility.cs

```diff
@@ -10,18 +13,45 @@ namespace Refresh.GameServer.Types;
 public enum Visibility
 {
     /// <summary>
-    /// User is okay with content being shown everywhere
+    /// User is okay with content being shown everywhere at all times
     /// </summary>
     [XmlEnum("all")]
     All = 0,
     /// <summary>
-    /// User only allows content to be shown in-game and on website 
+    /// User only allows content to be shown in-game and on the website to authenticated viewers 
     /// </summary>
     [XmlEnum("psn")] // Yes it says PSN, but in-game it is described as "users who are logged into PSN on the website"
-    Website = 1,
+    LoggedInUsers = 1,
     /// <summary>
     /// User only allows content to be shown in-game
     /// </summary>
     [XmlEnum("game")]
     Game = 2,
+}
+
+public static class VisibilityExtensions
+{
+    /// <summary>
+    /// Filters the passed object depending on the passed DataContext and intended visibility
+    /// </summary>
+    /// <param name="visibility">The intended visibility of the object</param>
+    /// <param name="visibility">The intended visibility of the object</param>
```

Comment:
Do you know who else has dementia?

```suggestion
    /// <param name="visibility">The intended visibility of the object</param>
```